cmd/sidecars/network-passt-binding/domain/configurator_test.go

```diff
@@ -347,6 +349,45 @@ var _ = Describe("pod network configurator", func() {
 
 			Expect(testMutator.Mutate(mutatedDomSpec)).To(Equal(mutatedDomSpec))
 		})
+	})
+	Context("should define memoryBacking for vhost-user", func() {
+		var testMutator *domain.PasstNetworkConfigurator
+		BeforeEach(func() {
+			networks := []vmschema.Network{*vmschema.DefaultPodNetwork()}
+			ifaces := []vmschema.Interface{{Name: "default", Binding: &vmschema.PluginBinding{Name: "passt"}}}
+
+			var err error
+			testMutator, err = domain.NewPasstNetworkConfigurator(ifaces, networks, domain.NetworkConfiguratorOptions{}, &netLinkStub{})
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should set shared memfd when clean domain", func() {
+			mutatedDomSpec, err := testMutator.Mutate(&domainschema.DomainSpec{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(mutatedDomSpec.MemoryBacking.Access.Mode).To(Equal("shared"))
+			Expect(mutatedDomSpec.MemoryBacking.Source.Type).To(Equal("memfd"))
```

Comment:
Please consider asserting the whole `MemoryBacking` struct against an expected one.